packages/react-jss/src/withStyles.js

```diff
@@ -1,316 +0,0 @@
-// @flow
```

Comment:
I am wondering if we should keep both interfaces until v11, it might make v10 adoption very slow, we should give people time to migrate to hooks. Especially considered React official statement is not to migrate classes to hooks, just for the sake of hooks, so people might be hesitative rewriting their entire code base and we end up everyone using v9.

I would not support  any maintenance of v9 though, because they can start using hooks API any time, it seems fully interoperable with a HOC and we can announce the HOC removal somewhere next year.

WDYT?

Reply:
Yeah, that's why I also now exported the `withStyles` as a named export. I think we should switch our docs to use the named export and deprecate the default export at some time later this year.

Reply:
I don't think we can remove `withStyles` any time soon, because unless react deprecates classes, we should support the API